delocate/delocating.py

```diff
@@ -526,42 +558,44 @@ def delocate_wheel(
         all_copied = {}  # type: Dict[Text, Dict[Text, Text]]
         wheel_dir = realpath(pjoin(tmpdir, 'wheel'))
         zip2dir(in_wheel, wheel_dir)
-        for package_path in find_package_dirs(wheel_dir):
-            lib_path = pjoin(package_path, lib_sdir)
-            lib_path_exists = exists(lib_path)
-            copied_libs = delocate_path(
-                package_path,
-                lib_path,
-                lib_filt_func,
-                copy_filt_func,
-                executable_path=executable_path,
-                ignore_missing=ignore_missing,
-            )
-            if copied_libs and lib_path_exists:
+        # Assume the package name from the wheel filename.
+        package_name = basename(in_wheel).split("-")[0]
+        lib_sdir = _decide_dylib_bundle_directory(
+            wheel_dir, package_name, lib_sdir
+        )
+        lib_path = pjoin(wheel_dir, lib_sdir)
+        lib_path_exists = exists(lib_path)
```

Comment:
Not your change, but I guess this should go above the following line, where it is first used.

Reply:
I thought `lib_path_exists` was put before `delocate_path` to tell if the `lib_path` existed before the call was made.  Maybe a better name would work.

Reply:
Well - let's leave for now.  But we aren't using the value for the `delocate_path` call.